web/src/server_events_dispatch.js

```diff
@@ -855,7 +855,6 @@ export function dispatch_normal_event(event) {
                 user_settings.enter_sends = event.value;
                 $(`.enter_sends_${!user_settings.enter_sends}`).hide();
                 $(`.enter_sends_${user_settings.enter_sends}`).show();
```

Comment:
I had a look around, and we don't seem to be using the `.enter_sends_true` and `.enter_sends_false` classes anymore (not to be confused with `enter_sends_true` arg being passed to the handlebars template). So we should probably get rid of these and any related CSS/JS in a separate commit, with a note for — b4b7188, as the initial commit which removes these classes.

Reply:
Found no CSS-related code for `.enter_sends_true` and `.enter_sends_false` classes.

Reply:
LGTM, thanks!